oeps/best-practices/oep-0066-bp-authorization.rst

```diff
@@ -0,0 +1,460 @@
+  it can be difficult to avoid accidentally changing other permissions with
+  a similar implementation.
+
+Systems/Protocols Overview Motivation
+-------------------------------------
+There are currently multiple systems/protocols that control authorization. 
+Determining which combination of systems/protocols are responsible for granting access 
+is a complex task made more complex by the lack of 
+unified documentation on the systems/protocols. This OEP aims to compile existing 
+knowledge and documentation into a central document that will give an overview of each 
+system/protocol. The aim is not to be the only source of information for each system/protocol, 
+but rather a starting point when learning about authorization within the Open edX codebase.
+
+Defined Terms
+*************
+
+Authorization (Authz)
+---------------------
+
+Authorization is the granting of permission of a certain user to perform specific operations in an application. A user can also delegate an application to be authorized to perform operations on their behalf without being logged in or authenticated, which is the basis of OAuth.
+
+.. note::
+  The definition of authorization found here is the same as that found in 
+  :doc:`OEP 42 <oep-0042-bp-authentication>`.
+  Credit for this definition belongs to the authors of OEP-42; Robert Raposa, Nimisha Asthagiri, and Julia Eskew.
+
+Authentication (Authn)
+----------------------
+
+Authentication is the verification of the identity of a user, which typically initiates at a “login” application point. Authentication is required whenever we need to identify a client/caller/user/etc.
+
+.. note::
+
+  Authentication is out of scope of this OEP. 
+  The definition is included here to clarify the difference between it and Authorization.
+  The definition comes from :doc:`OEP 42 <oep-0042-bp-authentication>`.
+  Credit for this definition belongs to the authors of OEP-42; Robert Raposa, Nimisha Asthagiri, and Julia Eskew.
+
+RBAC
+----
+Role Based Access Control. A system in which roles are assigned to a user 
+in order to grant that user permission to perform specific operations.
+
+There are multiple RBAC implementations in use within the Open edX codebase, 
+including, but not limited to, `edx-rbac`_` and `student_courseaccessrole`_.
+The implementations will be described in detail below.
+
+.. _student_courseaccessrole: https://github.com/openedx/edx-platform/blob/master/common/djangoapps/student/roles.py
+.. _edx-rbac: https://github.com/openedx/edx-rbac/tree/master
+
+Explicit Role
+-------------
+A role that is specifically assigned to a user with 
+the intent of providing the user permission to perform specific operations.
+
+Implicit Role
+-------------
+A "role" that is understood to belong to a user based 
+on data that is not role assignment data. 
+
+Implicit roles grant users permissions, but are not specifically assigned 
+to a user.
+
+System-Wide Role
```

Comment:
[Nit] The doc uses `System-Wide`, `System-wide`, and `system-wide`.  I prefer `System-wide`, at least in headers and starting sentences.